src/GraphQLSchemaLinter.php

```diff
@@ -49,6 +50,63 @@ public function getLinterConfigurationName() {
     return 'graphql-schema';
   }
 
+  private function checkVersion($version, $compare_to) {
+    $operator = '==';
+
+    $matches = null;
+    if (preg_match('/^([<>]=?|=)\s*(.*)$/', $compare_to, $matches)) {
+      $operator = $matches[1];
+      $compare_to = $matches[2];
+      if ($operator === '=') {
+        $operator = '==';
+      }
+    }
+
+    return version_compare($version, $compare_to, $operator);
+  }
+
+  public function getVersion() {
+    // Many node packages follow this same convention
+    list($err, $stdout, $stderr) = exec_manual('%C --version', $this->getExecutableCommand());
+
+    $matches = array();
+    $version = null;
+
+    if (preg_match('/^(\d\.\d\.\d)$/', $stdout, $matches)) {
+      $version = $matches[1];
+    } else {
+      $version = false;
+    }
```

Comment:
Should `$version` be `null` or `false` to indicate "undefined"? I think `null` makes more sense, in which case you can drop this `else` block.

Reply:
I will use `false` to be consistent with https://github.com/pinterest/arcanist-linters/blob/master/src/NodeExternalLinter.php#L83